// tbclient.AddPost.AddPostReqIdl
syntax = "proto3";

import "CommonReq.proto";

message AddPostReqIdl {
    message DataReq {
        CommonReq common = 1;
        string anonymous = 6;
        string can_no_forum = 7;
        string is_feedback = 8;
        string takephoto_num = 9;
        string entrance_type = 10;
        string vcode_tag = 16;
        string new_vcode = 18;
        string content = 19;
        string fid = 26;
        string v_fid = 28;
        string v_fname = 29;
        string kw = 30;
        string is_barrage = 31;
        string barrage_time = 32;
        string from_fourm_id = 44;
        string tid = 45;
        string is_ad = 51;
        string post_from = 55;
        string name_show = 58;
        string is_pictxt = 60;
        int32 show_custom_figure = 64;
        int32 is_show_bless = 67;
    }
    DataReq data = 1;
}
